resources/type-system/inference.md

```diff
@@ -696,6 +734,26 @@ constraint set `C` is define as follows:
 Note that the grounded constraint solution is a type, and hence may not contain
 occurences of the unknown type.
 
+#### Grounded constraint solution for a set of type variables
+
+The grounded constraint solution for a set of type variables `{X0, ..., Xn}`
+with respect to a constraint set `C`, with previous solution `{T0, ..., Tn}`, is
```

Comment:
As above, maybe say "and partial solution ..."?

Reply:
Done